scripts/gtcadis.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+import argparse
+import ConfigParser
+
+import numpy as np
+from pyne.mesh import Mesh
+from pyne.partisn import write_partisn_input, isotropic_vol_source
+from pyne.dagmc import discretize_geom, load
+from pyne import nucname
+from pyne.bins import pointwise_collapse
+
+
+config_filename = 'config.ini'
```

Comment:
Config files are rather old school.  Is there any way the config file could be in a more modern format, such as JSON or YAML?

Reply:
Perhaps old-fashioned, but supported natively within Python.  JSON is as well, but is perhaps a little more intimidating for users.

What YAML parser do you recommend?

Reply:
Yeah I agree that native support in python is desirable. I recommend `ruamel.yaml`, since it is pretty ubiquitous at this point.